### DISCUSSION TITLE
Using the Grid layout type for theme creators

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#94] [Topic Ideas]
Gutenberg 15.5 is introducing a new Grid layout type: https://github.com/WordPress/gutenberg/pull/49018

Currently, it's usable as an experimental Group block feature: https://github.com/WordPress/gutenberg/pull/49359

Once this is out of the experimental state, this will be a major layout feature for theme authors, and they'll need to know how to build with it. This is still early in development, so it'll be a while before it's ready for any sort of docs/tutorials. But, I wanted to go ahead and note it for a future post.

I'm super excited about this because I've just been wanting to build something like this layout for two years now with proper layout support instead of hacking Columns to make it happen:

![gb-gallery-w-quote](https://user-images.githubusercontent.com/1816309/229821976-a8f99150-0251-47da-a828-986c4f554903.jpg)

**Comment by bph** [comment]
This looks like some awesome feature!

**Comment by bph** [comment]
Stays as discussion for now. We can convert it to an issue once the feature is out of experiments and released in the Gutenberg plugin

**Comment by justintadlock** [comment]
Just noting that this is still experimental as of today and not going into WordPress 6.4. So we can still hold off on this for a bit.

**Comment by bph** [comment]
Discussion locked, conversation continues on issue

- #211
